pydoctor/test/test_sphinx.py

```diff
@@ -66,12 +66,20 @@ def inv_reader_nolog() -> sphinx.SphinxInventory:
 
 @pytest.fixture
 def inv_writer() -> InvWriter:
-    return InvWriter(logger=PydoctorLogger(), project_name='project_name')
+    return InvWriter(
+        logger=PydoctorLogger(),
+        project_name='project_name',
+        project_version='1.2.0.rc1',
```

Comment:
"1.2.0.rc1" is almost, but not quite, PEP 440 compliant (due to the dot before "rc"). I think it would be best to use a fully compliant version, but if you really want to test that the version is not processed in any way, it should be more explicit in its non-compliance.